config/envs/default.py

```diff
@@ -14,6 +14,9 @@ class DefaultConfig(object):
     FLASK_ROOT = str(Path(__file__).parent.parent.parent)
     FLASK_ENV = environ.get("FLASK_ENV")
 
+    ADMIN_SECRET = environ.get("ADMIN_SECRET")
```

Comment:
As discussed, as this is private API I don't think the `ADMIN_SECRET` is adding any value, please can we remove it and associated functionality?

Reply:
Removed